openslides_backend/action/actions/meeting/import_.py

```diff
@@ -549,9 +549,7 @@ def create_events(
                                 list_fields["add"][field] = value
                             else:
                                 fields[field] = value
-                        elif isinstance(
-                            model_field, (RelationListField, RelationField)
-                        ):
+                        elif isinstance(model_field, RelationListField):
```

Comment:
What's about RelationField?

Reply:
Does not work and is not needed here and lead to errors.

Reply:
The handling of the RelationField should still be in place.